src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerBinding.cs

```diff
@@ -84,5 +81,28 @@ public ParameterDescriptor ToParameterDescriptor()
                 CollectionName = this._documentCollectionLocation.CollectionName
             };
         }
+
+        internal static bool TryAndConvertToDocumentList(object value, out IReadOnlyList<Document> documents)
+        {
+            try
+            {
+                string valueAsString = value as string;
```

Comment:
small nit with C# 7 syntax:

```
if (value is IReadOnlyList<Document> docs) 
{
  documents = docs;
}
else if (value is string stringVal)
{
...
}
```

Reply:
@mamaso Thanks! I sometimes forget I can use C#7 in these repos 😄